lib/promiscuous/new_relic/instrumentation.rb

```diff
@@ -1,5 +1,31 @@
 require 'new_relic/agent/instrumentation/controller_instrumentation'
 
+module PromiscuousNewRelicInstrumentedClass
```

Comment:
can probably be pulled out into its own file, so promiscuous_black_hole doesn't need to instrument the UnitOfWork class it's not using